src/MathOptInterface.jl

```diff
@@ -62,6 +62,20 @@ Supported file types depend on the instance type.
 """
 function read! end
 
+"""
+    isempty(instance::AbstractInstance)
+
+Returns whether the `instance` has an objective set or has any variables or constraints its model.
```

Comment:
"Returns `false` if ..."

What about attributes? Is the instance still empty if you've set `ObjectiveSense()`?

Reply:
No, it is empty if copying into an empty instance is a no-op.